app-modules/engagement/src/Filament/ManageRelatedRecords/ManageRelatedEngagementRecords.php

```diff
@@ -274,6 +274,7 @@ public function table(Table $table): Table
                 CreateAction::make()
                     ->label('New Email or Text')
                     ->modalHeading('Create new email or text')
+                    ->authorize('create', $this->getOwnerRecord())
```

Comment:
This doesn't seem correct. I think you are specifically going to have to reference `Engagement` here.

@danharrin thoughts?

Reply:
This looks correct to me, the engagement is passed through automatically and this is an additional argument